src/puppeteer_utils.js

```diff
@@ -511,6 +512,58 @@ export const infiniteScroll = async (page, options = {}) => {
     }
 };
 
+/**
+ * Saves a full screenshot and HTML of the current page into a Key-Value store.
+ * @param {Object} page
+ *   Puppeteer <a href="https://pptr.dev/#?product=Puppeteer&show=api-class-page" target="_blank"><code>Page</code></a> object.
+ * @param {Object} [options]
+ * @param {String} [options.key=SNAPSHOT]
+ *   Key under which the screenshot and HTML will be saved. `.jpg` will be appended for screenshot and `.html` for HTML.
+ * @param {Number} [options.quality=50]
+ *   The quality of the image, between 0-100. Higher quality images have bigger size and require more storage.
+ * @param {Boolean} [options.saveScreenshot=true]
+ *   If true, it will save a full screenshot of the current page as a record with `key` appended by `.jpg`.
+ * @param {Boolean} [options.saveHtml=true]
+ *   If true, it will save a full HTML of the current page as a record with `key` appended by `.html`.
+ * @param {String} [options.storeName=null]
```

Comment:
storeName => keyValueStoreName

Reply:
Both `queueName` and `datasetName` is only used as a parameter name in constructor and field name for an internal function (BTW where it's clear from the context what it is). Here it's a public function in Utils, where it needs to be perfectly clear what kind of store this relates to.

Reply:
I searched the source code and `storeName` is used 

- in client - There you access it under `keyValueStores` so you don't need to long name there
- in SDK docs

I didn't find any reference for keyValueStoreName but I understand that `storeName` may not be clear in this context.

Reply:
In SDK it's only used as a param name of `KeyValueStore` class where it's clear what it is and can be renamed any time, as it's only parameter. In client it's used on key-value-stores namespace, where again it's clear.

Reply:
Let's not waste time here. It's a function called `Apify.utils.puppeteer.saveSnapshot` and `storeName` doesn't make it it clear what it is, without reading docs. That's a bad design. Let's rename it and move on.

Reply:
I agree with you, that is why I renamed it already.